Automattic/wp-calypso
# Gate Jetpack Sidebar Section with ABTest

#### Changes proposed in this Pull Request

* Add new ab test `jetpackSidebarSection` to control access to the new Jetpack Sidebar Section
* Switch granular control flags to be inclusive
* Remove features boot script

#### Testing instructions

1. Navigate to branch
2. Open the "ab tests" menu and verify you are **not** in the "showJetpackSidebarSection" <img width="759" alt="Screen Shot 2020-06-22 at 10 12 14 AM" src="https://user-images.githubusercontent.com/2810519/85316308-62327700-b471-11ea-8e20-e8a1a71ac717.png">
3. Use the "ab tests" menu to join the ab test
4. Verify the Jetpack Sidebar section appears as expected
